menpo/visualize/viewmatplotlib.py

```diff
@@ -196,18 +196,25 @@ def __init__(self, figure_id, new_figure, image):
         self.image = image
         self.axes_list = []
 
-    def render(self, interpolation='bilinear', alpha=1., render_axes=False,
-               axes_font_name='sans-serif', axes_font_size=10,
-               axes_font_style='normal', axes_font_weight='normal',
-               axes_x_limits=None, axes_y_limits=None, figure_size=(10, 8)):
+    def render(self, interpolation='bilinear', cmap=None, alpha=1.,
+               render_axes=False, axes_font_name='sans-serif',
+               axes_font_size=10, axes_font_style='normal',
+               axes_font_weight='normal', axes_x_limits=None,
+               axes_y_limits=None, figure_size=(10, 8)):
         import matplotlib.cm as cm
         import matplotlib.pyplot as plt
 
-        if len(self.image.shape) == 2:  # Single channels are viewed in Gray
-            plt.imshow(self.image, cmap=cm.Greys_r, interpolation=interpolation,
-                       alpha=alpha)
+        if len(self.image.shape) == 2:
+            if not cmap:
+                # Single channels are viewed in Gray by default
```

Comment:
I thought they were equivalent but `if cmap is None` is probably more explicit and better. I will change this.